src/components/AddItem/AddItem.component.jsx

```diff
@@ -44,11 +40,8 @@ const AddItem = props => {
   const addNewItemValue = event => {
     event.preventDefault();
 
-    // Clean Input to remove capitalization, punctuation, and spaces
-    const cleanInput = itemName
```

Comment:
I think the `cleanInput` function that includes punctuation and spaces is the one we want to include and remove lines from 43-44, rather than 47-51.

Reply:
Fixed